app/models/invitation_set.rb

```diff
@@ -4,7 +4,7 @@ class InvitationSet < ApplicationRecord
   belongs_to :person
   validates :person_id, presence: true
   has_many :invitations, dependent: :destroy
-  has_many :responses # no dependent destroy relation needed,
+  has_many :responses, dependent: :nullify # no dependent destroy relation needed,
```

Comment:
wow, dit is dat het ID op null wordt gezet?

Reply:
Ja. Van rubocop moet je nu een dependent clause definiëren. `nullify` betekent: als ik dit object verwijder, zet dan het ID dat naar dit object verwijst in alle andere objecten op null. Volgens mij was dat al de default (tenminste ik kan me niet voorstellen dat je in rails/postgres naar een id mag verwijzen dat niet bestaat).

Reply:
Maar nu is het dus expliciet.